app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from datetime import date
+
+
+from app.errors import NotVaccinatedError, OutdatedVaccineError
+from app.errors import NotWearingMaskError

```

Comment:
Use ( ) while importing multiple classes/modules https://github.com/mate-academy/py-lockdown-cafe/blob/master/checklist.md
They could be imported 3 from 1